lib/opensearch.rb

```diff
@@ -3,99 +3,138 @@
 # The OpenSearch Contributors require contributions made to
 # this file be licensed under the Apache-2.0 license or a
 # compatible open source license.
-#
-# Modifications Copyright OpenSearch Contributors. See
-# GitHub history for details.
-#
-# Licensed to Elasticsearch B.V. under one or more contributor
-# license agreements. See the NOTICE file distributed with
-# this work for additional information regarding copyright
-# ownership. Elasticsearch B.V. licenses this file to you under
-# the Apache License, Version 2.0 (the "License"); you may
-# not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-
-require 'opensearch/version'
+
+# This file is generated from the OpenSearch REST API spec.
+# Do not modify it by hand. Instead, modify the generator or the spec.
+
+require 'active_support/all'
 require 'opensearch/transport'
 require 'opensearch/api'
+Dir[File.expand_path('opensearch/api/**/*.rb', __dir__)].each { |f| require f }
```

Comment:
Should we `.sort` it to keep it stable?

Reply:
It used to have that but then Rubocop  complained after upgrading to Ruby 3.0. Ruby files in the api folders are not interdependent of each other unlike the old approach.